helm/reana/README.md

```diff
@@ -40,6 +40,8 @@ This Helm automatically prefixes all names using the release name to avoid colli
 | `ingress.annotations.kubernetes.io/ingress.class`        | Type of ingress controller                                                           | traefik                                         |
 | `ingress.annotations.traefik.frontend.entryPoints`       | Entrypoints allowed by the ingress controller                                        | "http,https"                                    |
 | `ingress.enabled`                                        | Create an ingress resource to access the REANA instance from outside the cluster     | true                                            |
+| `node_label_infrastructure`                              | Define the label which identifies the nodes where the infrastructure pods should run, e.g. `reana.io/system=infrastructure` | None |
+| `node_label_runtime`                                     | Define the label which identifies the nodes where the runtime pods should run, e.g. `reana.io/system=runtime` (note that if `runtime_namespace` is not set, this option will automatically enable it with the value `reana-runtime`) | None |
```

Comment:
Hmm, do we need to use different namespaces if we go for separate infrastructure labels and runtime labels? 

- If not, it would be good to keep the concepts separate, to allow people to use the default namespace (or the reana namespace) and still have an option to specify two labels to mark various nodes for various purposes;

- If the options are intertwined, perhaps we can advertise only one settings, e.g. please select namespace_infrastructure and namespace_runtime, and if they have different values, please label nodes etc etc.

Just trying to prevent any surprises if people would do `helm install -n mynamespace reana -f myvalues.yaml` manually...

Reply:
Solved as node labels and namespaces are not related anymore (the previous approach used [PodNodeSelector](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#podnodeselector) assigning the node selector to a Namespace, but this assumption was wrong because: (1) normally computer centres the creation of namespaces is not allowed (2) this feature wasn't working on Kind which means we cannot test it locally.